terraform/cross-account-IAM/pathfinder-to-analytical-platform-prod.tf

```diff
@@ -0,0 +1,74 @@
+data "aws_iam_policy_document" "pathfinder-prod-kiam-trust-chain" {
+  # KIAM trust chain to allow pods to assume roles defined below
+  statement {
+    principals {
+      type        = "AWS"
+      identifiers = [data.aws_iam_role.nodes.arn]
+    }
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+variable "pathfinder-prod-tags" {
+  type = map(string)
+  default = {
+    business-unit          = "HMPPS"
+    application            = "pathfinder"
+    is-production          = "true"
+    environment-name       = "prod"
+    owner                  = "pathfinder"
```

Comment:
The "owner" value should probably be 
```
"Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
```
That's consistent with other pathfinder stuff.

Reply:
👍